src/sentry/api/endpoints/integrations/sentry_apps/index.py

```diff
@@ -86,16 +86,27 @@ def post(self, request: Request, organization) -> Response:
         serializer = SentryAppSerializer(data=data, access=request.access)
 
         if serializer.is_valid():
-            data["redirect_url"] = data["redirectUrl"]
-            data["webhook_url"] = data["webhookUrl"]
-            data["is_alertable"] = data["isAlertable"]
-            data["verify_install"] = data["verifyInstall"]
-            data["allowed_origins"] = data["allowedOrigins"]
-            data["is_internal"] = data.get("isInternal")
-
-            creator = InternalCreator if data.get("isInternal") else Creator
```

Comment:
Consolidated the difference between these two creators into, well, one creator that handles the isInternal special casing.